wile.py

```diff
@@ -43,7 +44,12 @@ def wile(obj, directory_url, staging, account_key_path, new_account_key_size, ve
 
     if staging:
         directory_url = 'https://acme-staging.api.letsencrypt.org/directory'
-    account_key = get_or_gen_key(account_key_path, new_account_key_size)
+
+    try:
+        account_key = get_or_gen_key(account_key_path, new_account_key_size)
+    except ValueError as e:
```

Comment:
I don't see any real affect on that from d20af4badb06cdbd9416013a49185af57abd1c67, I still get `ValueError` as in #15. I would love to catch anything more high level.